### PR TITLE
Update kube-state-metrics maintainer list

### PR DESCRIPTION
**Pull request by mrueg**
/assign @lilic 
/assign @mrbobbytables

### CODE REVIEW

File: config/kubernetes/org.yaml [hunk 1]
```diff
@@ -1551,17 +1551,16 @@ teams:
   kube-state-metrics-admins:
     description: Admin access to the kube-state-metrics repo
     members:
-    - andyxning
```
Comment (lilic) [inline config/kubernetes/org.yaml:1554]: FYI @tariq1890 @brancz as andyxning was added to emeritus-approvers, so removing admin and write access makes sense. We can always add andyxning later on again if they decide to contribute more!

**Review by lilic — COMMENTED** [review]
lgtm from my side 🎉 

Thanks and welcome Manuel!

File: config/kubernetes/org.yaml (hunk 1 above)
Comment (tariq1890) [reply config/kubernetes/org.yaml:1554]: Sounds good to me!

**Pull request merged**